lectures/newtons_method.md

```diff
@@ -0,0 +1,469 @@
+---
+jupytext:
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.14.5
+kernelspec:
+  display_name: Python 3
+  language: python
+  name: python3
+---
+
+ ```{raw} html
+ <div id="qe-notebook-header" align="right" style="text-align:right;">
+         <a href="https://quantecon.org/" title="quantecon.org">
+                 <img style="width:250px;display:inline;" width="250px" src="https://assets.quantecon.org/img/qe-menubar-logo.svg" alt="QuantEcon">
+         </a>
+ </div>
+ ```
+
+# Using Newton’s Method to Solve Economic Models
+
+In addition to what’s in Anaconda, this lecture will need the following libraries:
+```{code-cell} ipython3
+---
+tags: [hide-output]
+---
+!pip install quantecon
+```
+
+## Overview
+
+Continuing from the the [Newton's Method lecutre](https://python.quantecon.org/newton_method.html), we are going to solve the multidimensional problem with `Jax`. More information about Jax can be found [here](https://python-programming.quantecon.org/jax_intro.html).
+
+We use the following imports in this lecture
+
+```{code-cell} python3
+import jax
+import jax.numpy as jnp
+import quantecon as qe
+import numpy as np
+from scipy.optimize import root
+```
+
+## The Two Goods Market Equilibrium
+
+Let's have a quick recall for this problem, a more detailed explanation and derivation can be found at [A Two Goods Market Equilibrium](https://python.quantecon.org/newton_method.html#two_goods_market).
+
+Assuming we have such market for two complementary goods where demand depends on the price of both components. 
+
+We label them $\text{good}_0$ and $\text{ good}_1$, with price vector $p = (p_0, p_1)$.
```

Comment:
$\text{good}_0$ -> good $0$

and the same for good 1   --- this is to be consistent with similar notation below.